core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -712,6 +712,15 @@ public NativeImageInvokerInfo build() {
                      * See https://github.com/quarkusio/quarkus/issues/30772 for more details.
                      */
                     nativeImageArgs.add("-H:+TrackNodeSourcePosition");
+                    /*
+                     * Starting with GraalVM/Mandrel 23.0, native-image strips debug info in a separate file by default.
+                     * Quarkus already does this, so for the time being we disable GraalVM/Mandrel's splitting as in
+                     * contrast to Quarkus it uses absolute paths which makes debugging harder especially for
+                     * in-container builds.
```

Comment:
I was too quick, sorry. There is no proof of this container issue. Even though the paths passed to `objcopy` are absolute, the links in the files end up being relative. So there seems to be no issue with upstream as far as I could tell. I'm not sure what the man page of `objcopy` alludes to, though. Anyway, perhaps we should use GraalVM's upstream version instead if it passes the failing integration tests?

Reply:
Still there is another difference between the GraalVM and the Quarkus approach. The former strips all info (including symbols which it keeps in a different file though) from the image while Quarkus strips only debug info.

Reply:
That's right. I agree that it should be OK to do slightly different debuginfo handling between 22.3 and 23.x.